style.css

```diff
@@ -1 +1,194 @@
 /*  Add your styling here */
+body{
+    margin: 0;
+    font-family: Arial, Helvetica, sans-serif;
+    display: grid;
+    grid-template-areas: 
+    "header"
+    "main"
+    "main2"
+    "footer";
+
+}
+a {
+    color: rgb(122, 0, 122);
+}
+
+
+.header1{
+    grid-area: header;
+    background-color: rgb(235, 195, 123);
+    position: fixed;
+    box-shadow: 1px 1px 4px rgba(0,0,0,.2);
+    width: 100%;
+    z-index: 10;
+    height:75px;
+}
```

Comment:
if you set this to use `display: flex`, you can set it to `flex-direction: column` on mobile, and then line the contents in a row on desktop with `flex-direction :row`

Reply:
but, it didn't  work with flex box...